postprocessing.py

```diff
@@ -9,38 +9,42 @@
 
 class BuildingLabeler(BaseTransformer):
     def transform(self, images):
-        labeled_images = []
+        return {'labeled_images': self._transform(images)}
+
+    def _transform(self, images):
         for i, image in enumerate(images):
             labeled_image = label(image)
```

Comment:
@taraspiotr will this work with inference batchsize > 1 ? I think the batches passed from pytorch _transform will be handled here and say batch of 8 is passed to label(image) image is .next on images generator if I understand correctly.